contents/blog/cookieless-analytics.md

```diff
@@ -0,0 +1,41 @@
+---
+date: 2021-08-13
+title: Cookieless Analytics - How To Track Website and App Users Without Third-Party Cookies
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: marcus-hyett
+featuredImage: ../images/blog/filename.png
+featuredImageType: full
+---
+
+Google has stated that they intend to deprecate the use of 3rd party cookies in 2023. But why is this important, and how will this affect product analytics?
+
+## What are 3rd party cookies?
+
+Cookies are small files stored on the user’s machine, helping you identify a repeat user to your product or website. This makes things like staying logged in over multiple days possible, but it also makes it possible to track whether the person who followed an ad yesterday came back today to purchase something.
+3rd party cookies are a particular flavour of cookie provided by an external service and not your website directly. This external service can be a tool or marketing product that you work with to optimize ads.
+
+## What happens when third-party cookies go away?
+
+When 3rd party cookies are deprecated, any external tools you use in your product or website to track users - either for analytics or marketing purposes - will cease to work correctly.
+Every visit to your website from the same user will look the same as a brand new user, making it difficult to attribute outcomes to your ad campaigns or understand whether a single person is avidly using your product or it’s a number of users using it casually.
+
+## What are the alternative solutions?
+
+It’s key to note that if people are logged into your product or website - you should still be able to track them (with any tool) - since you’ll be relying on first party cookies.
```

Comment:
This isn't true - if people today use amplitude/heap etc, they are using _3rd_ party cookies and their tracking will stop working.

Reply:
so this rule will have an even bigger impact than what this article states, which is great for our future

Reply:
I'm not sure this is the case @timgl  and I discussed this a few weeks ago...

If they have a logged in area of their website / product - they will very likely use sessions and thus 1st party cookies to administer that, I cannot see a reason why the js plugin or backend plugin for amplitude/heap would require a 3rd party cookie to identify the user - since the app itself has already identified them.

Perhaps someone who has a little more technical knowledge than me can chime in here (@neilkakkar?)